params/config.go

```diff
@@ -309,9 +309,9 @@ type ChainConfig struct {
 	CancunTime   *uint64 `json:"cancunTime,omitempty"`   // Cancun switch time (nil = no fork, 0 = already on cancun)
 	PragueTime   *uint64 `json:"pragueTime,omitempty"`   // Prague switch time (nil = no fork, 0 = already on prague)
 
-	BedrockBlock *big.Int `json:"bedrockBlock,omitempty"` // Bedrock switch block (nil = no fork, 0 = already on optimism bedrock)
-	RegolithTime *uint64  `json:"regolithTime,omitempty"` // Regolith switch time (nil = no fork, 0 = already on optimism regolith)
-
+	BedrockBlock     *big.Int `json:"bedrockBlock,omitempty"`     // Bedrock switch block (nil = no fork, 0 = already on optimism bedrock)
+	RegolithTime     *uint64  `json:"regolithTime,omitempty"`     // Regolith switch time (nil = no fork, 0 = already on optimism regolith)
+	L1ArchiveNodeRPC *string  `json:"l1ArchiveNodeRPC,omitempty"` // L1 Archive Node RPC URL (nil = no fork, 0 = already on optimism l1 archive node)
```

Comment:
Hmm this doesn't feel like the correct place necessarily because this is all of the hardfork info

Reply:
Regarding this comment, do you think the `Optimism *OptimismConfig `json:"optimism,omitempty"` would be a better place for it?

Reply:
It feels like its best for it to just be a CLI flag, these config structs are written to disk as part of the genesis info, I don't think it needs to be there